wd_helper.au3

```diff
@@ -699,12 +699,11 @@ Func _WD_HighlightElements($sSession, $vElements, $iMethod = Default)
 	If $iMethod < 0 Or $iMethod > 3 Then $iMethod = 1
 
 	If IsString($vElements) Then
-		ConsoleWrite('@@ Debug(' & @ScriptLineNumber & ') : $vElements = ' & $vElements & @CRLF & '>Error code: ' & @error & @CRLF) ;### Debug Console
 		$sScript = "arguments[0].style='" & $aMethod[$iMethod] & "'; return true;"
 		$sResult = _WD_ExecuteScript($sSession, $sScript,  __WD_JsonElement($vElements), Default, $_WD_JSON_Value)
 		$iErr = @error
 
-	ElseIf IsArray($vElements) Then
+	ElseIf IsArray($vElements) And UBound($vElements) > 0 Then
```

Comment:
`IsArray($vElements) And` is not needed.
`UBound($vElements)` do the same at once.

Reply:
Just 
```AutoIt
ElseIf UBound($vElements) Then
```
It's enough/

Reply:
or... you are preparing script for `map` ?
As `UBound()` can be used also for `map` thus yours `IsArray()` usage ?

Reply:
Yes, having `IsArray()` in the code will ensure that only variables of the correct type are processed in this section of code.

Reply:
I keep forgetting about maps.
So this part is ok.